### ISSUE TITLE
Enable HTTPS/SSL

### ISSUE THREAD

**Issue by helenahalldiniths** [Patlenlix/CrimeDatabase#39]
fifth part of security
https://www.youtube.com/watch?v=HLSmjZ5vN0w&list=PLVApX3evDwJ1d0lKKHssPQvzv2Ao3e__Q&index=15

**Comment by helenahalldiniths** [comment]
This issue is depending on issue #38 to be compleated.

**Comment by helenahalldiniths** [comment]
I think this part can be added later on. It is a little complicated and not part of the "g" demands, however the video above is a good resource and this shouldn't be that hard to implement later on.
